### PR TITLE
lesson 10

### PR DESCRIPTION
**Pull request by AndreiChernukho**

### CODE REVIEW

File: src/com/tms/lesson10/task2/Main.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    static final String PATHREADFILE = "src/com/tms/lesson10/task2/resources/textAll.txt";
+    static final String PATHWRITEFILE = "src/com/tms/lesson10/task2/resources/text.txt";
+    static final String PATHWRITEFILE2 = "src/com/tms/lesson10/task2/resources/text2.txt";
+    static final String USER_FIRST = "Собака: ";
+    static final String USER_SECOND = "Кошка: ";
+
+    public static void main(String[] args) throws IOException {
+
+        boolean isFirst = true;
+        String textAll = readFile(PATHREADFILE).toString();
```
Comment (gr00shik) [inline src/com/tms/lesson10/task2/Main.java:17]: what happen if file is too large?

**Review by gr00shik — COMMENTED** [review]
ok

File: src/com/tms/lesson10/task1/Main.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    private static String USER_FILE_NAME ="user.dat";
+
+    public static void main(String[] args) {
+        User user = new User("Vanja", 16, Sex.MALE);
+        serialization(user);
+        deserialization();
+    }
+
+    private static void serialization(User user) {
```
[the remark this answers is not in the document]
Comment (AndreiChernukho) [reply src/com/tms/lesson10/task1/Main.java:18]: fix

File: src/com/tms/lesson10/task2/Main.java (hunk 1 above)
Comment (AndreiChernukho) [reply src/com/tms/lesson10/task2/Main.java:17]: fix

**Pull request merged**